koku/cost_models/serializers.py

```diff
@@ -398,9 +400,9 @@ def to_representation(self, cost_model_obj):
             display_data = self._get_metric_display_data(cost_model_obj.source_type, metric.get("name"))
             metric.update(
                 {
-                    "label_metric": display_data.label_metric,
-                    "label_measurement": display_data.label_measurement,
-                    "label_measurement_unit": display_data.label_measurement_unit,
+                    "label_metric": display_data.get("label_metric", ""),
+                    "label_measurement": display_data.get("label_measurement", ""),
+                    "label_measurement_unit": display_data.get("label_measurement_unit", ""),
```

Comment:
I don't think we should be passing back empty string here.
Seems more like we want to check that these values are not None before we update the metric and if any are None it seems like a 500 because the constant should have these items.

Reply:
added